ui/src/pages/AllStories/reducer.ts

```diff
@@ -19,14 +23,21 @@ export type Action =
   | { type: "HANDLE_CHECKED"; e: React.ChangeEvent; story: StoryView }
   | { type: "SET_ORDERING"; order: "asc" | "desc"; orderBy: string }
   | { type: "SET_TABLE_DATA"; data: StoryView[] }
-  | { type: "SET_TAB_VALUE"; newValue: number };
+  | { type: "SET_TAB_VALUE"; newValue: number }
+  | { type: "SET_CHANGED_VISIBILITY"; data: StoryView[] }
```

Comment:
consider changing the input variable name for readability (ie. instead of "data", "newVisibility")

Reply:
hmm I think we should just keep it all as data for consistency? The type should be explanatory for what the action does